sled-storage/src/dataset.rs

```diff
@@ -34,16 +36,16 @@ pub const M2_BACKING_DATASET: &'static str = "backing";
 cfg_if! {
     if #[cfg(any(test, feature = "testing"))] {
         // Tuned for zone_bundle tests
-        pub const DEBUG_DATASET_QUOTA: usize = 1 << 20;
+        pub const DEBUG_DATASET_QUOTA: u64 = 1 << 20;
```

Comment:
Can these be `ByteCount`s?

Reply:
I don't think so, because the `ByteCount` impls are not `const` right now. Wish they were though!